rosdep/python.yaml

```
@@ -5104,6 +5104,17 @@ python3-bson:
       packages: [bson]
   rhel: ['python%{python3_pkgversion}-bson']
   ubuntu: [python3-bson]
+python3-build:
+  alpine: [py3-build]
+  arch: [python-build]
+  debian: [python3-build]
+  fedora: [python3-build]
+  gentoo: [dev-python/build]
+  osx: [python-build]
+  rhel:
+    '9': [python3-build]
+  ubuntu:
+    jammy: [python3-build]
```

Generally it is better to mark things as '*' so that they are automatically used for newer versions, and then to "null" out the older ones that aren't supported.  So in this case this would be better:

```suggestion
  ubuntu:
    '*': [python3-build]
    focal: null
```

---

Fixed